addons/website_forum/static/src/js/website_forum.js

```diff
@@ -112,7 +116,7 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
             var $textarea = $(textarea);
             var editorKarma = $textarea.data('karma') || 0; // default value for backward compatibility
             if (!$textarea.val().match(/\S/)) {
-                $textarea.val('<p><br/></p>');
+                $textarea.val('<p/>');
```

Comment:
is it working with futur editor for v12 ? I thing that without the <b'/>, user cannot have the focus between the <p> and </p>

Reply:
If I understood well yes, there is no issue with focus.
https://drive.google.com/file/d/1t6KVC2HOBjKIgWonF25kDv3mqfN01ozI/view?usp=sharing
Is what your are referring to?

Reply:
JKE said with the futur editor since we are reverting back to the v12 one (normally merged in the upcoming days) ;) I guess we'll have to see then but even now I don't see a real reason to remove the BR ;)